instructure/lti2_reference_tool_provider
# Registration and launch

Closes _PLAT-2295_, _PLAT-2296_

**Test Plan**
- Clone repo and follow	README. Verify the tool is able to run locally and specs pass.
- Run the tool and install in Canvas. Verify tool proxy registration succeeds in Canvas.
- Verify the tool proxy is persisted in the tool's database.
- Add the tool to a module item and verify basic lti launch is successful.
- Verify the basic lti launch responds with 404 if the signature sent my the TC is invalid.
- Verify tool includes `status=failure` if Canvas does not give a 201 in response to the tool proxy create post request.
- Verify comments are helpful. 
- Check for spelling and grammar errors
